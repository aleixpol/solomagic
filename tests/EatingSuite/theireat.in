\ref something0
\mt somethingsomething andia somethingsomething

\ref something1
\mt somethingsomething nandia somethingsomething
