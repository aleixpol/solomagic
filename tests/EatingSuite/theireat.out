\ref something0
\mt somethingsomething andia somethingsomething
\ma somethingsomething angria somethingsomething

\ref something1
\mt somethingsomething nandia somethingsomething
\ma somethingsomething nandia somethingsomething
